paper.md

```diff
@@ -85,8 +72,37 @@ interpolation between tabulated values. To satisfy the need for a more
 accurate implementation of this integral, we contributed
 `scipy.stats.studentized_range` [@PRstudentized_range], a class that
 evaluates the cumulative distribution function and many other functions of the
-distribution. Both statsmodels and Pingouin have since adopted this class to
-perform studentized range distribution calculations.
+distribution.
+
+To address numerical difficulties, ratios in the integrand are implemented by
+exponentiating the difference in the logarithms of the individual terms.
+Consequently, `scipy.stats.studentized_range` works well for $\nu$ through at
+least $10^5$. The integrand is written in Cython and integrated using
+`scipy.integrate.nquad`, resulting in evaluation times on the order of 5 ms
```

Comment:
Based on
![image](https://user-images.githubusercontent.com/6570539/170086905-1f514533-ffc2-43ee-bcdb-f9f2ab5c8654.png)

but omitting the specifications of all the computers it's been times on (as that would seem out of place in this short paper).